src/renderers/dom/shared/devtools/ReactDOMUnknownPropertyDevtool.js

```diff
@@ -63,12 +63,20 @@ if (__DEV__) {
       null
     );
 
+    // Suggest an event name correction if possible
     warning(
       registrationName == null,
-      'Unknown event handler property %s. Did you mean `%s`?',
+      'Unable to register unsupported event handler property %s. Did you mean %s?',
       name,
       registrationName
     );
+
+    // Otherwise at least make it clear that the attributes will be filtered out
+    warning(
+      standardName != null || registrationName != null,
+      'Unable to assign unsupported DOM property %s.',
+      name
+    );
   };
 }
```

Comment:
I updated the warning copy hear to make it clearer what's going on. Happy to change this.